dom.bs

```diff
@@ -1186,33 +1187,70 @@ for discussion).
 
  <li><p>Initialize <var>event</var>'s {{Event/currentTarget}} attribute to <var>object</var>.
 
+ <li><p><a>Find</a> listener type matches for <var>object</var>.
+
  <li>
-  <p>For each <var>listener</var> in <var>listeners</var>, whose <b>removed flag</b> is unset, run
+  <p>If <var>event</var>'s <a>found listener flag</a> is unset, run
```

Comment:
This flag is checked as part of "invoke" only, and unset at the end of "invoke". Given that "invoke" is not invoked recursively, how does this work?